hot-reload-devtools/src/main/kotlin/org/jetbrains/compose/devtools/sidecar/DtAttachedSidecar.kt

```diff
@@ -5,170 +5,81 @@
 
 package org.jetbrains.compose.devtools.sidecar
 
-import androidx.compose.animation.AnimatedContent
-import androidx.compose.animation.EnterTransition
-import androidx.compose.animation.ExitTransition
 import androidx.compose.animation.animateContentSize
-import androidx.compose.animation.core.tween
-import androidx.compose.animation.fadeIn
-import androidx.compose.animation.fadeOut
-import androidx.compose.animation.scaleIn
-import androidx.compose.animation.togetherWith
-import androidx.compose.foundation.ExperimentalFoundationApi
-import androidx.compose.foundation.clickable
 import androidx.compose.foundation.layout.Arrangement
 import androidx.compose.foundation.layout.Column
-import androidx.compose.foundation.layout.Row
 import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.layout.size
 import androidx.compose.runtime.Composable
-import androidx.compose.runtime.LaunchedEffect
-import androidx.compose.runtime.getValue
-import androidx.compose.runtime.mutableStateOf
-import androidx.compose.runtime.remember
-import androidx.compose.runtime.setValue
 import androidx.compose.ui.Alignment
-import androidx.compose.ui.ExperimentalComposeUiApi
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
-import androidx.compose.ui.input.pointer.PointerEventType
-import androidx.compose.ui.input.pointer.onPointerEvent
 import androidx.compose.ui.window.WindowState
-import kotlinx.coroutines.delay
 import org.jetbrains.compose.devtools.Tag
-import org.jetbrains.compose.devtools.tag
-import org.jetbrains.compose.devtools.theme.DtColors
+import org.jetbrains.compose.devtools.sendAsync
+import org.jetbrains.compose.devtools.theme.DtImages
 import org.jetbrains.compose.devtools.theme.DtPadding
 import org.jetbrains.compose.devtools.theme.DtShapes
 import org.jetbrains.compose.devtools.theme.DtSizes
 import org.jetbrains.compose.devtools.theme.DtTitles.COMPOSE_HOT_RELOAD_TITLE
 import org.jetbrains.compose.devtools.widgets.DtComposeLogo
+import org.jetbrains.compose.devtools.widgets.DtImage
+import org.jetbrains.compose.devtools.widgets.DtIconButton
 import org.jetbrains.compose.devtools.widgets.animateReloadStatusColor
+import org.jetbrains.compose.devtools.widgets.applyIf
 import org.jetbrains.compose.devtools.widgets.dtBackground
-import org.jetbrains.compose.reload.core.HotReloadEnvironment.devToolsAnimationsEnabled
+import org.jetbrains.compose.reload.core.HotReloadEnvironment
 import org.jetbrains.compose.reload.core.WindowId
+import org.jetbrains.compose.reload.orchestration.OrchestrationMessage
 
 private val cornerShape = if (devToolsUseTransparency) DtShapes.RoundedCornerShape else DtShapes.SquareCornerShape
-private val useAnimatedTransitions = devToolsUseTransparency && devToolsAnimationsEnabled
 
 @Composable
 fun DtAttachedSidecarWindow(
```

Comment:
I think the wording 'detached' makes sense to differentiate the special case, but I constantly notice that my brain needs to resolve the 'attached' wording when reading the code. 

Suggestion:
We have one 'happy path', 'regular' case (which is currently called) 'attached'. 
We have one 'special case' which is called 'detached'. 

How about just differentiating it by calling the detached code 'detached' and regular code would be unclassified. 

e.g. here: 
`DtSidecarWindow` vs `DtDetachedSidecarWindow`